src/Common/src/System/Collections/Generic/LowLevelComparer.cs

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+// LowLevelComparer<T> emulates the desktop type System.Collections.Generic.Comparer<T>
+//
+
+using System;
+using System.Collections;
+
+namespace System.Collections.Generic
+{
+    internal sealed class LowLevelComparer<T> : IComparer<T>
```

Comment:
Why is this type necessary?  Why is just Comparer<T> (also in this assembly) insufficient?